test/conftest.py

```diff
@@ -5,7 +5,6 @@
 
 import py
 import pytest
-
```

Comment:
It's not an error.
If flake8 reacts to this, it must be configured.

https://peps.python.org/pep-0008/#imports
```
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
```

Reply:
Fixed with .isort.conf:
`[settings]
known_local_folder=utils`

Reply:
1. Will it use config master/.flake8 ? - Yes
2. gofmt added (goimports is buggy)
3. isort is checked on CI.